### ISSUE TITLE
make client-watch stops watching after syntax error

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#4864]
```
[1] SyntaxError: /Users/marten/devel/git/galaxy/client/galaxy/scripts/mvc/library/library-model.js: Unexpected token, expected , (92:68)
[1] error Command failed with exit code 1.
[1] yarn run gulp watch exited with code 1
```

**Comment by dannon** [comment]
My first thought was "Well, `run.sh --reload` has worked like this for 10 years now...", but I fixed it anyway with #4867.

Thanks @martenson

**Comment by martenson** [comment]
I was not aware of `run.sh --reload`. However the previous client watcher did not behave like this so thanks for fixing this regression.
